pipeline-modules/deployment-service/aws/lambda-gateway.yaml

```diff
@@ -0,0 +1,152 @@
+provisioner: aws
+name: lambda-gateway
+version: 1
+revision: 1
+displayName: "AWS Lambda deployment"
+description: "deploy lambda functions with codepipes and serverless"
```

Comment:
change to "Code Pipes"

Reply:
Is the fact that we are using "serverless" material to the consumer?

Reply:
not exactly unless we state what this template is capable of doing